packages/hardhat-cannon/src/tasks/inspect.ts

```diff
@@ -0,0 +1,65 @@
+import path from 'path';
+import { task } from 'hardhat/config';
+import { TASK_INSPECT } from '../task-names';
+import loadCannonfile from '../internal/load-cannonfile';
+import { getChartDir, getAllDeploymentInfos } from '@usecannon/builder';
+import { NetworksConfig } from 'hardhat/types';
+import { DeploymentInfo } from '@usecannon/builder/dist/types';
+import chalk from 'chalk';
+const { red, bold, gray, green, cyan } = chalk;
+
+task(TASK_INSPECT, 'Inspect the deployments in a cannon package')
+  .addFlag('json', 'Output as JSON')
+  .addOptionalParam('file', 'TOML definition of the chain to inspect', 'cannonfile.toml')
+  .setAction(async ({ file, json }, hre) => {
+    const filepath = path.resolve(hre.config.paths.root, file);
+    const def = loadCannonfile(hre, filepath);
+    const { name, version } = def;
+    const chartsDir = getChartDir(hre.config.paths.cannon, name, version);
+    const deployInfo = await getAllDeploymentInfos(chartsDir);
+
+    if (json) {
+      console.log(JSON.stringify(deployInfo, null, 2));
+    } else {
+      if (deployInfo?.deploys) {
+        console.log(green(bold(`The package ${name}:${version} has the following deployments...`)));
```

Comment:
this seems somewhat unprofessional... with the ...s at the end... maybe more like 
```
=========== ${name}:${version} ===========
```